docs/websites/varnish/use-varnish-with-nginx-to-serve-cached-wordpress-content-over-ssl-on-debian-8.md

```diff
@@ -336,26 +341,85 @@ Before configuring nginx we have to install PHP-FPM. FPM is short for FastCGI Pr
         listen = /var/run/php5-fpm.sock
         ~~~
 
+4.  Verify that the owner of the FPM pool is 'www-data'. The `user` and `group` directives have to be set to `www-data` like this:
```

Comment:
Since these settings are set like this by default, what's the purpose of these two steps?